OpenRA.Mods.Common/Widgets/Logic/ModBrowserLogic.cs

```diff
@@ -159,5 +194,41 @@ void LoadMod(ModMetadata mod)
 				Game.InitializeMod(mod.Id, null);
 			});
 		}
+
+		static bool IsModInstalled(ModMetadata mod)
+		{
+			var manifest = new Manifest(mod.Id);
+			var folders = manifest.Folders.Where(f => f.Contains("^")).Select(f => f.Substring(f.IndexOf("^", StringComparison.Ordinal)));
+
+			var filesList = new List<string>();
+			foreach (var folder in folders)
+			{
+				var path = Platform.ResolvePath(folder).Replace("/", "\\");
```

Comment:
This hard-codes the directory separator to the Windows only `\` which breaks everywhere else so please change this to
```csharp
var path = Platform.ResolvePath(folder);
```